flake.nix

```diff
@@ -116,7 +116,6 @@
               pkgs.cargo-machete
               pkgs.cargo-nextest
               pkgs.cargo-watch
-              pkgs.rnix-lsp
```

Comment:
👀

Reply:
It's been marked as deprecated in `nixpkgs`, so removing for now.